### PR TITLE
Update new services for AWS

### PR DESCRIPTION
**Pull request by tvqphuoc01**
Introduce additional services for AWS:
1. OpenSearch
2. CodeArtifact
3. DevOps Guru
4. Network Firewall
5. Proton
6. AppConfig
7. Amazon Managed Grafana
8. Amazon Managed Prometheus
9. AWS Managed Workflow for Apache Airflow

### CODE REVIEW

**Review by gabriel-tessier — CHANGES_REQUESTED** [review]
look fine except the proton file which is 500x500 should be 256x256

**Comment by tvqphuoc01** [comment]
> look fine except the proton file which is 500x500 should be 256x256

updated the size of Proton. Thanks

**Review by filipeaaoliveira — APPROVED** [review]

File: diagrams/aws/analytics.py [hunk 1]
```diff
@@ -104,6 +104,10 @@ class ManagedStreamingForKafka(_Analytics):
     _icon = "managed-streaming-for-kafka.png"
 
 
+class Opensearch(_Analytics):
+    _icon = "opensearch.png"
+
+
```
Comment (gabriel-tessier) [inline diagrams/aws/analytics.py:110]: Should be OpenSearch with the file open-search.png

when checking on my AWS console the OpenSearch top page display the name with a S in capital case. 

"Amazon OpenSearch Service"

File: diagrams/aws/devtools.py [hunk 2]
```diff
@@ -20,6 +20,10 @@ class Cloud9(_Devtools):
     _icon = "cloud9.png"
 
 
+class Codeartifact(_Devtools):
+    _icon = "codeartifact.png"
+
+
```
Comment (gabriel-tessier) [inline diagrams/aws/devtools.py:26]: Same here for the name the aws console page for code artifact is display like this: 

**AWS CodeArtifact**

File: diagrams/aws/management.py [hunk 3]
```diff
@@ -72,6 +84,10 @@ class ControlTower(_Management):
     _icon = "control-tower.png"
 
 
+class DevopsGuru(_Management):
+    _icon = "devops-guru.png"
+
```
Comment (gabriel-tessier) [inline diagrams/aws/management.py:89]: Here too the AWS name is written DevOps with capital O

Amazon DevOps Guru

**Review by gabriel-tessier — CHANGES_REQUESTED** [review]
Left comments for the icons naming.

File: diagrams/aws/devtools.py (hunk 2 above)
Comment (tvqphuoc01) [reply diagrams/aws/devtools.py:26]: hmm i think currently we currently use this kind of naming for all devtool services so I don't this one should be change.
image

**Review by gabriel-tessier — APPROVED** [review]

**Pull request merged**